ops/scripts/deployer.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
-RETRIES=${RETRIES:-20}
+set -eou
```

Comment:
What does this do?

Reply:
I've updated the main comment with the answer